src/VoteView.js

```diff
@@ -2,25 +2,51 @@ import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import axios from 'axios';
 import CandidateVotes from './Candidate';
+import { Link } from 'react-router-dom';
+import { Button } from './Candidate';
+
 const url =
   'http://ec2-13-209-5-166.ap-northeast-2.compute.amazonaws.com:8000/api/candidates';
 
-function VoteView() {
+function LoginoutButton({ isLogined, handleLogoutButton }) {
```

Comment:
```suggestion
function LoginoutButton({ isLogined, onClick }) {
```

이 함수는 프롭스를 이벤트 이름으로 지어야지. 상위 컴포넌트에서 
```jsx
<LoginoutButton onClick={handleLogoutButtonClick} />
```

이렇게 쓸 수 있도록!